lib/stat_tracker.rb

```diff
@@ -40,4 +40,36 @@ def highest_total_score
     end
     return output.away_goals + output.home_goals
   end
+
+  def percentage_ties
+    tie_games = @game_teams.find_all do |game|
+      game.game_result.include?("TIE")
+    end
+    results = tie_games.length.to_f / @game_teams.length.to_f
+    results.round(2)
+  end
+
+  def count_of_games_by_season
+    ids = []
+    only_ids = []
+      @games.find_all do |game|
+        ids = game.season_id
+        only_ids << ids
+      end
+    uniq_ids = only_ids.uniq
+    count_games = []
```

Comment:
I'm wondering again if this collector needs to be established here or not, since it doesn't seem to be used at this scope later on in your code